### ISSUE TITLE
firefox startup: AAKlist telling me to subscribe

### ISSUE THREAD

**Issue by worldsdream** [reek/anti-adblock-killer#2194]
This is the message I get: It seems that you have not subscribed or disabled AakList. Subscribe or Disable this alert.

Firefox Stable
Greasemonkey
Adguard extension
Reek js for greasemonkey
Mac

**Comment by hackel** [comment]
I have the same problem.  I have subscribed to the list with uBlock Origin:
https://raw.githubusercontent.com/reek/anti-adblock-killer/master/anti-adblock-killer-filters.txt
This has been happening for months now.  I'm not sure why the automatic detection is failing, but it is quite frustrating!

FF52, uBO 1.10.4.

**Comment by yochaigal** [comment]
I have the same issue with Chrome 55 for Linux and ublock origin.

**Comment by SebastianRasch** [comment]
Yeah this started here as well now, macOS/Safari. The problem is that there is a blank in the list link and that’s probably why it doesn’t recognise that I’m already subscribed.
As you can see on the screenshot, the blank is after the word "anti":
![bildschirmfoto 2017-02-19 um 19 01 48](https://cloud.githubusercontent.com/assets/22617697/23104862/f4252584-f6d5-11e6-848b-60fc6a3c9426.png)

**Comment by Dracozny** [comment]
known issue for a long time that never got fixed. except for that space issue, @Superkloton mentions. 
You can turn off the subscription notification @ https://reek.github.io/anti-adblock-killer/#settings

**Comment by zax-ftw** [comment]
Just wanted to say that this is still bugged even though it was mentioned to be fixed in 10.0 here: https://github.com/reek/anti-adblock-killer/issues/890

**Comment by libove** [comment]
As per @zax-ftw's reference above this exact thing is now happening also to me presently. 
Chrome 64-bit  64.0.3282.167 on Windows 10 Enterprise 10.0.16299 Build 16299 (1709 Fall Creators Update).
Within TamperMonkey I see the update URL as:
https://greasyfork.org/scripts/735-anti-adblock-killer-reek/code/Anti-Adblock%20Killer%20%7C%20Reek.user.js
When I click to manually update, it says that it updates successfully, but it also shows that the last update (at the moment) was 16d ago. In settings the Script Update check interval is 1d, and the Externals Update interval is 1w. 

I see that I can set the logging level from its default Error to Debug, but I don't see where those logs would go. If someone would be so kind as to point out where the debug log would become available, I will run in Debug log mode for a while and see if I can get a clue of where the "It seems you have not subscribed or disabled ..." message is being caused and whether it's a display bug (but the updates really are happening) or whether it really means something bigger is wrong. Thank you.

**Comment by TraceyC77** [comment]
This is still an issue for me with Firefox 76.0 and uBlock Origin 1.26.2
After getting the warning, I checked my filter list in uBlock Origin and saw that AakList is present with the URL

moz-extension://ad97292d-743c-2f40-8855-cc7ebebe6c16/asset-viewer.html?url=https://raw.github.com/reek/anti-adblock-killer/master/anti-adblock-killer-filters.txt

**Comment by liamengland1** [comment]
https://github.com/reek/anti-adblock-killer/wiki

**Comment by worldsdream** [comment]
@TraceyC77 remove reek from your lists. It’s not being maintained.
